test/powershell/Modules/Microsoft.WinGet.CommandNotFound/Microsoft.WinGet.CommandNotFound.Tests.ps1

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+Describe "Microsoft.WinGet.CommandNotFound" -tags "CI" {
+    BeforeAll {
+        if (Get-Module Microsoft.WinGet.CommandNotFound) {
+            Remove-Module Microsoft.WinGet.CommandNotFound
+        }
+    }
+
+    Context "Windows only" {
+        It "Should import the module correctly" -Skip:(-not $IsWindows) {
+            Import-Module -Name Microsoft.WinGet.CommandNotFound
```

Comment:
since this is the only test which imports the module, you could do this:
```powershell
try {
            $module = Import-Module -Name Microsoft.WinGet.CommandNotFound -PassThru
            $module.Name | Should -BeExactly 'Microsoft.WinGet.CommandNotFound'
            $module.Version | Should -Match '^1.0.4$'
}
finally {
            Get-Module -Name Microsoft.WinGet.CommandNotFound | Remove-Module -Force
}
```

and then remove the `AfterAll`